Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG APP_INSIGHTS_AGENT_VERSION=3.2.4
+ARG APP_INSIGHTS_AGENT_VERSION=3.4.8
```

Comment:
version 3.4.9 appears to be the latest which some of our projects are already upgraded to.

Reply:
Ok, I will upgrade to 3.4.9